<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationManager</class>
 <widget class="QDialog" name="ConfigurationManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab_clipboard">
        <attribute name="title">
         <string>&amp;Clipboard</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,0,0">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>&amp;Interval to check clipboard (milliseconds):</string>
               </property>
               <property name="buddy">
                <cstring>spinInterval</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinInterval">
               <property name="minimum">
                <number>50</number>
               </property>
               <property name="maximum">
                <number>1000000</number>
               </property>
               <property name="value">
                <number>1000</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Callback &amp;script:</string>
               </property>
               <property name="buddy">
                <cstring>lineEditScript</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditScript"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Save &amp;formats:</string>
               </property>
               <property name="buddy">
                <cstring>lineEditFormats</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditFormats">
               <property name="text">
                <string>text/plain text/html image/bmp image/x-inkscape-svg-compressed</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>&amp;Synchronize</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QCheckBox" name="checkBoxClip">
                  <property name="text">
                   <string>clipboard &gt;&gt; history (&amp;1)</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBoxCopyClip">
                  <property name="text">
                   <string>clipboard &gt;&gt; selection (&amp;2)</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBoxSel">
                  <property name="text">
                   <string>selection &gt;&gt; history (&amp;3)</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBoxCopySel">
                  <property name="text">
                   <string>selection &gt;&gt; clipboard (&amp;4)</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxConfirmExit">
             <property name="text">
              <string>Confirm application e&amp;xit</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_general">
        <attribute name="title">
         <string>Clipboard &amp;History</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Maximum &amp;number of items in history list:</string>
               </property>
               <property name="buddy">
                <cstring>spinBoxItems</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBoxItems">
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="value">
                <number>200</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>N&amp;umber of items in tray menu:</string>
               </property>
               <property name="buddy">
                <cstring>spinBoxTrayItems</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBoxTrayItems">
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>&amp;Format display priority:</string>
               </property>
               <property name="buddy">
                <cstring>lineEditPriority</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditPriority">
               <property name="text">
                <string>image/bmp image/x-inkscape-svg-compressed text/plain text/html</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>External &amp;editor command (%1 is file to edit):</string>
               </property>
               <property name="buddy">
                <cstring>lineEditEditor</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditEditor">
               <property name="text">
                <string>gedit %1</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_commands">
        <attribute name="title">
         <string>Co&amp;mmands</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QTableWidget" name="tableCommands">
             <column>
              <property name="text">
               <string>Enable</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Name</string>
              </property>
              <property name="toolTip">
               <string>Label in context menu (use ampersand before character you want to be quick access key)</string>
              </property>
              <property name="whatsThis">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Command</string>
              </property>
              <property name="toolTip">
               <string>Command (%1 is contents of selected items)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Input</string>
              </property>
              <property name="toolTip">
               <string>Send content of selected items to standard input</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Output</string>
              </property>
              <property name="toolTip">
               <string>Save the output in items</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Separator</string>
              </property>
              <property name="toolTip">
               <string>Command output separator for splitting into multiple items</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Match</string>
              </property>
              <property name="toolTip">
               <string>Allow command only on matched items (regular expression)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Wait</string>
              </property>
              <property name="toolTip">
               <string>Open action dialog instead of executing the command immediately</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Icon</string>
              </property>
              <property name="toolTip">
               <string>Icon filename</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Shortcut</string>
              </property>
              <property name="toolTip">
               <string>Command shortcut (e.g. Ctrl-X)</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="pushButtoAdd">
               <property name="text">
                <string>A&amp;dd</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonRemove">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Remove [Del]</string>
               </property>
               <property name="shortcut">
                <string>Del</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonUp">
               <property name="text">
                <string>Move Command &amp;Up</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonDown">
               <property name="text">
                <string>Move Command D&amp;own</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_appearance">
        <attribute name="title">
         <string>&amp;Appearance</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="1">
          <widget class="QSplitter" name="verticalLayout_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="opaqueResize">
            <bool>true</bool>
           </property>
           <property name="childrenCollapsible">
            <bool>true</bool>
           </property>
           <widget class="QGroupBox" name="verticalLayout_6">
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <property name="margin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>History &amp;item format (HTML):</string>
                 </property>
                 <property name="buddy">
                  <cstring>plainTextEdit_html</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPlainTextEdit" name="plainTextEdit_html">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="plainText">
                  <string>&lt;table class=\&quot;item\&quot;&gt;&lt;tr&gt;&lt;td class=\&quot;number\&quot;&gt;%1&lt;/td&gt;&lt;td class=\&quot;text\&quot;&gt;%2&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QGroupBox" name="verticalLayout_7">
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <property name="horizontalSpacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>&amp;Style sheet (CSS):</string>
                 </property>
                 <property name="buddy">
                  <cstring>plainTextEdit_css</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPlainTextEdit" name="plainTextEdit_css">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="plainText">
                  <string>/* http://doc.qt.nokia.com/stylesheet-examples.html */
/* font */
.text, .number, #clipboardBrowser, #searchBar, QPlainTextEdit {
    font-family: Envy Code R, Consolas, Bitstream Vera Sans Mono, monospace;
}

#clipboardBrowser, #searchBar {
    border: 0px solid transparent;
}

QMainWindow {
}

#clipboardBrowser {
}

#searchBar {
}

#findLabel {
    font-weight: bold;
}

/* item text */
.text {
}

/* item number */
.number {
    padding-right: 1ex;
    color: #444;
}

.format {
    font-size: 6pt;
    font-style: italic;
}

.format.current {
    text-decoration: underline;
}

/* matched text */
.em {
    background-color: yellow;
    color: black;
}

QToolTip, QScrollBar {
}

QListView::item {
    border: 3px solid transparent;
}

QListView::item:alternate {
    background-color: rgba(0,240,255,30);
}

QListView::item:selected {
    background-color: rgba(255,255,100,150);
}

QListView::item:hover {
    border: 2px dashed #888;
}</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>spinInterval</tabstop>
  <tabstop>lineEditScript</tabstop>
  <tabstop>lineEditFormats</tabstop>
  <tabstop>checkBoxCopyClip</tabstop>
  <tabstop>checkBoxCopySel</tabstop>
  <tabstop>spinBoxItems</tabstop>
  <tabstop>spinBoxTrayItems</tabstop>
  <tabstop>lineEditPriority</tabstop>
  <tabstop>lineEditEditor</tabstop>
  <tabstop>checkBoxClip</tabstop>
  <tabstop>checkBoxSel</tabstop>
  <tabstop>checkBoxConfirmExit</tabstop>
  <tabstop>tableCommands</tabstop>
  <tabstop>pushButtoAdd</tabstop>
  <tabstop>pushButtonRemove</tabstop>
  <tabstop>pushButtonUp</tabstop>
  <tabstop>pushButtonDown</tabstop>
  <tabstop>plainTextEdit_html</tabstop>
  <tabstop>plainTextEdit_css</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigurationManager</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigurationManager</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
